tests/cypress/e2e/for-teacher_page/class_page/second_teachers_invites.cy.js

```diff
@@ -1,83 +1,78 @@
 import {loginForTeacher, logout} from "../../tools/login/login.js"
 import { goToProfilePage } from "../../tools/navigation/nav";
-import { createClass } from "../../tools/classes/class";
+import { createClass, navigateToClass } from "../../tools/classes/class";
 
 const secondTeachers = ["teacher2", "teacher3"]
-const invitesTable = body => body.find("#invites-block table")
 
+// NOTE: These test steps must execute in sequence. Each of them depends on the previous
+// test having executed, in the order they are listed in the file.
+// They cannot be made independent until https://github.com/hedyorg/hedy/issues/4804 is resolved
 describe("Second teachers: invitations", () => {
+  // Before all: create a single class
+  let className;
   before(() => {
     loginForTeacher();
-    createClass()
+    className = createClass()
   });
 
   it(`Invites ${secondTeachers.length} second teachers: by username`, () => {
     loginForTeacher();
-    cy.get(".view_class").first().click();
+    navigateToClass(className);
```

Comment:
Right! I forgot that we had this function. I also selfishly wanted this test to work for my PR to be merged hahaha